### ISSUE TITLE
IDEA: Change all the two-char functions

### ISSUE THREAD

**Issue by iounpaladin** [DennisMitchell/jelly#11]
You could change the two-char atoms to either overloads of other atoms, or one-char atoms. I assume there isn't already 256 used bytes.

Sorry for typos, I'm on mobile.

**Comment by ConorOBrien-Foxx** [comment]
How do you suggest overloading? Considering that atoms have fixed arity, you might be referring to type overloading. If you are, I don't see how this would save many bytes, in the long run.

**Comment by erikkonstas** [comment]
I'm afraid it's too late now :-1:... and yes, I think all 256 bytes were being used before you even asked for this.

**Comment by zacharyacutey** [comment]
What about the two-char nilads? Those could possibly be copied into the unused nilad bytes.

**Comment by GolfingSuccess** [comment]
I think that once they are assigned their two-byte names, they cannot change, because it will break each and every PPCG answer using them **at all**. Maybe newer nilads could go to one-byte names.

**Comment by zacharyacutey** [comment]
That's why I said **copied**.

**Comment by GolfingSuccess** [comment]
Nah, that would be impractical and wasteful. Newer nilads could use that space! Or, make a second version of the language to transfer them and use 2 bytes for newer ones.

**Comment by zacharyacutey** [comment]
Well, at least nilads like pi should be a single-byte as well as a double-byte.

**Comment by GolfingSuccess** [comment]
See my edited message.
